### DISCUSSION TITLE
Need to fix bytes to bits

### DISCUSSION THREAD

**Discussion by ilnarb** [ai/nanoid#513] [General]
Need to fix bytes to bits in in all texts:
* A tiny (124 **bytes**), secure
* 118 **bytes** (minified and brotlied)
* and so on

**Comment by ai** [comment]
Why?

It is Size Limit metric. It is in bytes.

**Reply by ilnarb** [reply]
oh, is it not about id length?

**Reply by ai** [reply]
Not, it is the size of JS file of the library
